### ISSUE TITLE
bug: HA - mqtt topics hardcoded

### ISSUE THREAD

**Issue by ikkemaniac** [rvdbreemen/OTGW-firmware#17]
The state topics sent to home assistant discovery are hard coded, they should be set the to ```<topic prefix>``` variable

For example
https://github.com/rvdbreemen/OTGW-firmware/blob/379e7358c4c52b71fdab894246ed27f7d5ad2e54/data/mqttha.cfg#L3

```
"state_topic": "OTGW/fault"
```
should be:

```
"state_topic": topic_prefix_variable +"/fault"
```

This is also true for the home assistant discovery topic. It defaults to ```homeassistant``` but a user has the option to change this. A config option/var should be available to set it.

**Comment by rvdbreemen** [comment]
Yep, this can be fixed. Give me time, or do a PR.

**Comment by rvdbreemen** [comment]
@ikkemaniac fix in dev stream of v0.7.8 
- hardcoded state topic now flexible
- added homeassistant prefix at the same time
- adding setting in webUI

**Comment by ikkemaniac** [comment]
nice, i'll have a look and test next weekend!
